internal/eval/eval.go

```diff
@@ -79,7 +79,7 @@ func evalPING(args []string, store *dstore.Store) []byte {
 	return b
 }
 
-// evalECHO returns the argument passed by the user 
+// evalECHO returns the argument passed by the user
```

Comment:
seems like linting issues got fixed when my editor ran format. seems like we don't enforce go fmt in CI